package.json

```diff
@@ -40,7 +40,8 @@
     "test:storybook": "test-storybook"
   },
   "dependencies": {
-    "@forgerock/javascript-sdk": "^4.1.2",
+    "@forgerock/javascript-sdk": "../forgerock-javascript-sdk/dist/packages/javascript-sdk",
```

Comment:
Once we get a beta of the SDK released, we can change this to point to the beta instead.